akka-remote/src/main/scala/akka/remote/RemoteMetricsExtension.scala

```diff
@@ -51,18 +55,38 @@ private[akka] class RemoteMetricsOff extends RemoteMetrics {
   override def logPayloadBytes(msg: Any, payloadBytes: Int): Unit = ()
 }
 
+/**
+ * INTERNAL API
+ */
+private object RemoteMetricsOn {
+  val LogFrameSizeExceedingConfigKey = "akka.remote.log-frame-size-exceeding"
+}
+
 /**
  * INTERNAL API
  */
 private[akka] class RemoteMetricsOn(system: ExtendedActorSystem) extends RemoteMetrics {
 
-  private val logFrameSizeExceeding: Int = system.settings.config.getBytes(
-    "akka.remote.log-frame-size-exceeding").toInt
   private val log = Logging(system, this.getClass)
+
+  private val logFrameSizeExceeding: Int = {
+    if (system.settings.config.getString(LogFrameSizeExceedingConfigKey) == "off") {
```

Comment:
this can be removed, it can't happen, because if "off" then `RemoteMetricsOff` is used instead of `RemoteMetricsOn`

(and it's anyway missing a `toLowerCase`)